packages/fluentui/docs/src/examples/components/Form/Usage/FormExampleComponents.tsx

```diff
@@ -0,0 +1,61 @@
+import * as React from 'react';
+import { Form, Input, Text } from '@fluentui/react-northstar';
+
+const items = [
+  {
+    name: 'pizza',
+    key: 'Capricciosa',
+    label: 'Capricciosa',
+    value: 'capricciosa',
+  },
+  {
+    name: 'pizza',
+    key: 'Prosciutto',
+    label: 'Prosciutto',
+    value: 'prosciutto',
+    disabled: true,
+  },
+  {
+    name: 'pizza',
+    key: 'Pepperoni',
+    label: 'Pepperoni',
+    value: 'pepperoni',
+  },
+  {
+    name: 'pizza',
+    key: 'Custom',
```

Comment:
Let's remove this one as for an example it's too complicated

Reply:
Done